lighthouse-core/audits/dobetterweb/uses-http2.js

```diff
@@ -43,7 +43,7 @@ class UsesHTTP2Audit extends Audit {
 
       // Filter requests that are on the same host as the page and not over h2.
       const resources = networkRecords.filter(record => {
-        const requestHost = new URL(record._url).host;
+        const requestHost = record._parsedURL && record._parsedURL.host;
```

Comment:
this is the only place we were using `new URL` for network records over `parsedURL`, the only other place where we use `new URL` in a loop is for the anchor tags which I think is less likely to have hundreds/thousands of `target=_blank` elements, so maybe it's ok?

I'm not sure what the overlap of duplicate URLs would be for caching